src/CalcViewModel/Common/TraceLogger.cpp

```diff
@@ -303,12 +304,21 @@ namespace CalculatorApp
         TraceLoggingCommon::GetInstance()->LogLevel2Event(StringReference(EVENT_NAME_VARIABLE_SETTING_CHANGED), fields);
     }
 
-    void TraceLogger::LogGraphSettingsChanged(GraphSettingsType settingType)
+    void TraceLogger::LogGraphSettingsChanged(GraphSettingsType settingType, String ^ settingValue)
     {
         auto fields = ref new LoggingFields();
         fields->AddString(StringReference(CALC_MODE), StringReference(GRAPHING_MODE));
         fields->AddInt16(L"SettingType", static_cast<int16>(settingType));
+        fields->AddString(L"SettingValue", settingValue);
 
         TraceLoggingCommon::GetInstance()->LogLevel2Event(StringReference(EVENT_NAME_GRAPH_SETTINGS_CHANGED), fields);
     }
+    void TraceLogger::LogGraphTheme(Platform::String ^ graphTheme)
```

Comment:
nit: add space above to match the style used in this file

Reply:
Also, I wish this file was more consistent about using "Platform::String" or "String". We should be consistent, at least for the code that's changing in this PR.

Reply:
Style of the project is to omit the namespace in cpp files